src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a target="_blank" href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="/"
+              class="nav-link is-active"
+            >apple</a>
```

Comment:
fix attributes indent in all cases

```suggestion
            <a
              href="/"
              class="nav-link is-active"
            >
              apple
            </a>
```